config/config.go

```diff
@@ -185,6 +192,12 @@ func InitConfig() *Config {
 		viper.SetDefault(fmt.Sprintf("Monitorable.Github.%s.InitialMaxDelay", variant), DefaultInitialMaxDelay)
 	}
 
+	for variant := range variants["Stripe"] {
```

Comment:
Add comment prefix as other monitorable
```suggestion
  // --- Stripe Configuration ---
	for variant := range variants["Stripe"] {
```